src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -443,7 +443,18 @@
                                         <StackPanel Orientation="Horizontal"
                                                     ContextMenu="{StaticResource FileContextMenu}"
                                                     Tag="{Binding DataContext, RelativeSource={RelativeSource Mode=FindAncestor,AncestorType={x:Type te:SectionControl}}}">
-                                            <ui:OcticonImage Icon="file_code" Foreground="{DynamicResource GitHubVsWindowText}"/>
+                                            <ui:OcticonImage Icon="file_code" Foreground="{DynamicResource GitHubVsWindowText}">
+                                                <ui:OcticonImage.Style>
+                                                    <Style TargetType="ui:OcticonImage" BasedOn="{StaticResource OcticonImage}">
+                                                        <Style.Triggers>
+                                                            <DataTrigger Binding="{Binding Status}" Value="Removed">
+                                                                <Setter Property="Foreground" Value="{DynamicResource GitHubDeletedFileIconBrush}"/>
+                                                                <!--<Setter Property="Foreground" Value="Red"/>-->
```

Comment:
I'm having a bit of trouble with the binding here. I know I have the trigger right, as I'm able to see the effect of the value `Red` in the Visual Studio designer, but at runtime neither the DynamicResource binding nor the value `Red` seem to have an effect.